src/preciceMap.cpp

```diff
@@ -126,6 +126,21 @@ std::vector<int> setupMesh(precice::SolverInterface &interface, const aste::Mesh
                               edgeMap.at(Edge{b, c}),
                               edgeMap.at(Edge{c, a}));
   }
+
+  VLOG(1) << "Mesh Setup: 4) Quadrilaterals";
+  for (auto const &quadrilateral : mesh.quadrilaterals) {
+    const auto a = vertexIDs[quadrilateral[0]];
+    const auto b = vertexIDs[quadrilateral[1]];
+    const auto c = vertexIDs[quadrilateral[2]];
+    const auto d = vertexIDs[quadrilateral[3]];
+
+    interface.setMeshQuad(meshID,
```

Comment:
Don't forget to extend `gather_unique_edges` to make sure that these edges exist.